builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -246,12 +260,22 @@ node()  {
 	}
 }
 
+def getRequiredData(){
```

Comment:
Can you move this function to util module and make use in all workflows as it is a common method for all, we can avoid the code duplication

Reply:
moved the util module